packages/react-text/src/components/Text/useText.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { makeMergeProps, getSlots, resolveShorthandProps } from '@fluentui/react-compose/lib/next/index';
+import { TextProps } from './Text.types';
+
+const mergeProps = makeMergeProps();
+
+/**
+ * Define the render function. Given the state of a text, renders it.
+ */
+export const renderText = (state: TextProps) => {
```

Comment:
Should we move `renderText` into a separate file?

Reply:
Moved 👍